.ci/jobs/pull-request.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+- job:
+    name: "elastic+rally+pull-request"
+    display-name: "elastic / rally # pull-request"
+    description: "CI testing for rally pull requests"
+    scm:
+      - git:
+          refspec: "+refs/pull/*:refs/remotes/origin/pr/* +refs/heads/*:refs/remotes/origin/*"
+          branches:
+            - "${ghprbActualCommit}"
+    triggers:
+      - github-pull-request:
+          org-list:
+            - elastic
+          allow-whitelist-orgs-as-admins: true
+          trigger-phrase: '.*run\W+tests.*'
+          github-hooks: true
+          status-context: tests
```

Comment:
should this match the name of the Jenkins cluster? e.g. `elasticsearch-ci`? (it's possible it doesn't matter, would be nice to know what this is about).

Reply:
It's just the name of the PR check. If you take a look at this PR's checks, for example, you'll see one called "tests".